src/groups/special_euclidean.jl

```diff
@@ -619,28 +582,15 @@ function lie_bracket!(G::SpecialEuclidean, Z, X, Y)
     return Z
 end
 
-"""
```

Comment:
Same question here, we could keep the docs attached to the signature.

Reply:
Same answer.